docs/broker/README.md

```diff
@@ -112,26 +112,30 @@ kubectl -n default get broker default
 #### Manual Setup
 
 In order to setup a `Broker` manually, we must first create the required
-`ServiceAccount` and give it the proper RBAC permissions. This setup is required
+`ServiceAccount`s and give them the proper RBAC permissions. This setup is required
```

Comment:
Now that you are editing this file... can you fix the cosmetic:

the **implmentation** may change at any time, absolutely no guarantees are made about the **implmentation**

Also, inside Implementation, Namespace, you will have to add the steps of creating the eventing-broker-filter service account, and the RBAC.

Reply:
Now that is fresh, there is a broker_trigger.md file in knative/docs, with this manual steps as well. It'd be great if you can create a follow up PR on docs with this same changes...

Reply:
Made the changes here.

Once this gets in, I'll immediately open a PR on docs to update it their too. 

Maybe we should just delete this file? And rely on docs only?

Reply:
Yes, I think it'd be better to remove this one. Just check if it has the same content. I think the docs one doesn't have the implementation, but I think it's fine to skip that section.

Reply:
OK. In a follow up PR (after I submit the changes to docs), I'll pair this down to a link to docs and implementation details.

Reply:
Sounds good!